src/stan/variational/advi.hpp

```diff
@@ -334,6 +520,14 @@ namespace stan {
 
         // draw more samples from posterior and write on subsequent lines
         if (out_stream_) {
```

Comment:
I feel like I'm missing something: why is there an `out_stream_` and `print_stream_` here for this if statement? I could understand one or the other.